langchain/utilities/vertexai.py

```diff
@@ -0,0 +1,73 @@
+"""Utilities to init Vertex AI."""
+from typing import TYPE_CHECKING, Optional
+
+if TYPE_CHECKING:
+    from google.oauth2.service_account import Credentials
+
+
+def _create_credentials_from_file(
+    json_credentials_path: Optional[str],
+) -> Optional["Credentials"]:
+    """Creates credentials for GCP.
+
+    Args:
+        json_credentials_path: The path on the file system where the
+        credentials are stored.
+
+    Returns:
+        An optional of Credentials or None, in which case the default
+        will be used.
+    """
+
+    from google.oauth2 import service_account
+
+    credentials = None
+    if json_credentials_path is not None:
+        credentials = service_account.Credentials.from_service_account_file(
+            json_credentials_path
+        )
+
+    return credentials
+
+
+def raise_vertex_import_error() -> ValueError:
+    sdk = "google-cloud-aiplatform>=1.25.0"
+    raise ImportError(
+        "Could not import VertexAI. Please, install it with " f"pip install {sdk} "
```

Comment:
It's better to put quotes around the full package name+version. Othervise the shell can interpret `>` as output redirection.

Reply:
done